client/my-sites/site-monitoring/metrics-tab.tsx

```diff
@@ -296,12 +298,12 @@ export const MetricsTab = () => {
 			<SiteMonitoringLineChart
 				timeRange={ timeRange }
 				title={ __( 'Server performance' ) }
-				subtitle={ __( 'Requests per minute and average server response time' ) }
+				subtitle={ __( 'Requests per second and average server response time' ) }
 				data={ formattedData as uPlot.AlignedData }
 				series={ [
 					{
 						fill: 'rgba(6, 117, 196, 0.1)',
-						label: __( 'Requests per minute' ),
+						label: __( 'Requests per second' ),
```

Comment:
We deliberately chose `Requests per minute` because smaller sites won't have many requests per second.

What inspired you to change it to `Requests per second?`

Reply:
I started this PR trying to avoid decimals in the number of requests and unify the algorithm to transform requests per second given a time slot into a different unit.

This PR does not calculate the requests per minute, nor per second, It calculates requests per the time slot returned by the API. It multiplies `_meta.resolution * value`. The resolution in multiple days is 2 hours, so in that case, it would be the number of requests per 2 hours, which displays higher numbers in the scale and the actual number of requests that the site has received.

- Currently, I have another alternative that represents requests per minute: https://github.com/Automattic/wp-calypso/pull/80770 and probably is more aligned on what we want, a fixed unit. Requests per minute.

The difference is if each point in the chart represents a slot of one minute or if it represents the actual time slot returned by the API (2 hours).